reference/enterprise-api-ref/export-import-api.markdown

```diff
@@ -0,0 +1,232 @@
+---
+layout: default
+title: Import & Export API
+published: true
+tags: [reference, enterprise, API, import, export]
+---
+
+Import & Export API allows to transfer Mission portal data between hubs.
+
```

Comment:
Import & Export API provides users the ability to transfer Mission Portal data between hubs.

Reply:
done